browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -230,6 +233,30 @@ class MarkdownNoteDetail extends React.Component {
     }
   }
 
+  handleFullScreenButton (e) {
+    const currentScreenState = !Object.assign({}, this.state).fullScreen
```

Comment:
gotcha, I ended up having to wrap the show/hide code inside the callback of the setState. It wasn't being updated properly so I read the react docs and found " React does not guarantee that the state changes are applied immediately.".